src/app/reference-data/reference-data.component.ts

```diff
@@ -13,18 +13,40 @@ import { Observable } from 'rxjs';
 export class ReferenceDataComponent {
   referenceDataList$: Observable<RefrenceDataResponseDto>;
   referenceDataList: ReferenceDataModel[];
+  selectedRefDataType: string;
 
-  constructor(private modalService:NgbModal, private apiService: RealestateApiService){}
+constructor(private modalService:NgbModal, private apiService: RealestateApiService){}
   
-  ngOnInit(): void {
+ngOnInit(): void {
   this.apiService.getAllReferenceData().subscribe((referenceDataList$: any) => {
-    this.referenceDataList = referenceDataList$.typologiesList;
-  })
+    this.referenceDataList = referenceDataList$.realEstateTypesList;
+    this.selectedRefDataType = '2';
```

Comment:
The (change) method isn't called when you set the type to 2? Not sure if it is, but if yes, we don't need line 22. 
We just need to set the type to 2, and then the dropdownFilterChanged / dropDownFilter method will take care of the rest.

Reply:
If we remove line 22, the page don't show nothing because the (change) method just work after we interact with the select.